bin/cutting_edge

```diff
@@ -1,57 +1,87 @@
 #!/usr/bin/env ruby
 
-require File.expand_path('../../lib/cutting_edge/app.rb', __FILE__)
 require 'yaml'
 require 'rufus-scheduler'
+require 'optparse'
 
-REFRESH_SCHEDULE = '1h'
-
-config = <<YAML
-github:
-  gollum:
-    gollum:
-      api_token: secret
-      email: false
-    gollum-lib:
-      api_token: secret
-      locations: [gemspec.rb]
-      dependency_types: [runtime, development]
-      email: gollum@mydependencymonitoring.com
-  singingwolfboy:
-    flask-dance:
-      language: python
-      email: false
-  rust-lang:
-    crates.io:
-      language: rust
-      api_token: secret
-gitlab:
-  cthowl01:
-    team-chess-ruby:
-      locations: [Gemfile]
-      api_token: secret
-YAML
+module CuttingEdge
+  REFRESH_SCHEDULE = '1h'
+end
 
 options = {
   :port => 4567,
-  :bind => '0.0.0.0',
+  :bind => '0.0.0.0'
 }
 
-store = Moneta.new(:Memory)
-CuttingEdge::App.set(:store, store)
+opts = OptionParser.new do |opts|
+  opts.banner = 'CuttingEdge is a dependency monitoring application.
+  
+  Usage:
+      cutting_edge [options] [projects]
+      
+  Arguments:
+      [projects]                     Path to the YAML file which defines the projects to be monitored. If not specified, projects.yml in the current working directory is used.
```

Comment:
Indentation seems off.

Reply:
Yeah it seems weird, but oddly it's the indentation that's required to get everything to align in the terminal output 😕